helm-system-packages-brew.el

```diff
@@ -0,0 +1,214 @@
+				    (format format-string name desc)))
+				  (split-string descriptions"\n") "\n" ))
+  (cons names descriptions)))
+
+(defun helm-system-packages-brew-init ()
+  "Cache package lists and create Helm buffer."
+  (unless (and helm-system-packages--names helm-system-packages--descriptions)
+    (helm-system-packages-brew-refresh))
+  ;; TODO: We should only create the buffer if it does not already exist.
+  ;; On the other hand, we need to be able to override the package list.
+  ;; (unless (helm-candidate-buffer) ...
+  (helm-init-candidates-in-buffer
+      'global
+    (if helm-system-packages-show-descriptions-p
+        helm-system-packages--descriptions
+      helm-system-packages--names)))
+
+(defun helm-system-packages-brew-refresh ()
+  "Refresh the package list."
+  (interactive)
+  (setq helm-system-packages--source-name "brew source")
+  (setq helm-system-packages-column-width
+        (or helm-system-packages-brew-column-width
+            helm-system-packages-column-width))
+    (let ((res (helm-system-packages-brew-cache)))
+      (setq helm-system-packages--names (car res)
+            helm-system-packages--descriptions (cdr res)))
+    (setq helm-system-packages--display-lists nil))
+
+(defun helm-system-packages-brew-info (_candidate)
+  "Print information about the selected packages.
+
+With prefix argument, insert the output at point.
+Otherwise display in `helm-system-packages-buffer'."
+  (let* ((descriptions (json-read-from-string (with-temp-buffer
+						(apply 'call-process "brew" nil t nil "info" "--json=v1" (helm-marked-candidates)) 
+						(buffer-string))))
+	 desc-list
+	 pkg-desc-alist
+	 str
+	 pkg
+	 (i 0))
+    (dolist (pkg (helm-marked-candidates) nil)
+      (setq pkg-desc-alist (aref descriptions i))
+      (setq str (concat "* Description: " (alist-get 'desc pkg-desc-alist) "\n"
+	      "* Version: "(alist-get 'stable (alist-get 'versions pkg-desc-alist)) "\n"
+	      "* URL: "(alist-get 'homepage pkg-desc-alist) "\n"  "\n"
+	      "* Dependencies:\n" "   " (mapconcat 'identity (alist-get 'dependencies pkg-desc-alist) "\n   ") "\n\n"
+	      "* Optional dependencies:\n" "   "(mapconcat 'identity (alist-get 'optional_dependencies pkg-desc-alist) "\n   ") "\n\n"
+	      "* Options:\n" (mapconcat (lambda (pkg-option)
+					(concat (alist-get 'option pkg-option)  "\n"
+						"    " (alist-get 'description pkg-option)  "\n"))
+				      (alist-get 'options pkg-desc-alist) "\n")
+	      "\n\n"
+	      "* Caveats: " (alist-get 'caveats pkg-desc-alist) "\n"))
+     (add-to-list 'desc-list `(uninstalled (,pkg . ,str)))
+     (setq i (+ i 1)))
+    (helm-system-packages-show-information desc-list)))
+
+(defun helm-system-package-brew-browse-url (_candidate)
+   (let* ((descriptions (json-read-from-string (with-temp-buffer
+						(apply 'call-process "brew" nil t nil "info" "--json=v1" (helm-marked-candidates)) 
+						(buffer-string)))))
+    (helm-system-packages-browse-url (mapcar (lambda(pkg)
```

Comment:
Missing space before `(pkg)`.

Reply:
Should be fixed by https://github.com/tuedachu/helm-system-packages/pull/1.